src/viam/examples/dial/example_dial.cpp

```diff
@@ -68,7 +68,5 @@ int main() {
         std::cout << "got generic client named " << gc->name() << std::endl;
     }
 
-    robot->close();
```

Comment:
Per previous PR, this call resulted in a double `close`.

Reply:
Ah yeah I did file a ticket for this one earlier: https://viam.atlassian.net/browse/RSDK-6212.

Reply:
Thanks for digging that up.